config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -17,12 +17,11 @@ presubmits:
       containers:
         - image: gcr.io/k8s-testimages/kubekins-e2e:v20210601-ea6aa4e-master
           command:
-            - runner.sh
-            - bash
-            - -c
-          args:
-            # TODO: stop depending on custom timeout
-            - 'make test KUBE_RACE=-race KUBE_TIMEOUT=--timeout=240s'
+            - make
```

Comment:
not strictly related, but normalizing how we specify the command vs the CI job and upcoming replacement rootless config